File: FizzBuzz.c
#include<stdio.h>
#include<conio.h>
void main(){
    int i;
    for(i = 1 ; i<=100 ; i++){
        if(i % 15 == 0){
            printf("FizzBuzz\t");
        }
        else if(i % 5 == 0){
            printf("Buzz\t");
        }
        else if(i % 3 == 0){
            printf("Fizz\t");
        }
        else{
            printf("%d\t",i);
        }
    }
}

File: FizzBuzz.php
<?php 
    for($i = 1 ; $i <=100 ; $i++)
        if($i%15 == 0)
            echo "<br>FizzBuzz<br>";
        elseif ($i%3) 
            echo "<br>Fizz<br>";
        elseif  ($i%5)
            echo "<br>Buzz<br>";
        else    
            echo $i;    
?>

            
